@navikt/core/css/darkside/date.darkside.css

```diff
@@ -1,222 +1,206 @@
-  box-shadow: var(--a-shadow-focus);
-}
+  .navds-date__field {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    position: relative;
+  }
 
-@supports not selector(:focus-visible) {
-  .navds-date .rdp-button:not(.rdp-day_selected, [disabled]):focus,
-  .navds-date .navds-date__month-button:not(.rdp-day_selected, [disabled]):focus {
-    box-shadow: var(--a-shadow-focus);
+  .rdp-day_selected,
+  .navds-monthpicker__month--selected {
+    color: var(--ax-text-accent-contrast);
+    background: var(--ax-bg-accent-strong-pressed);
+    cursor: pointer;
   }
-}
 
-.navds-date .rdp-button.rdp-day_selected:not([disabled]):focus-visible,
-.navds-date .navds-date__month-button.rdp-day_selected:not([disabled]):focus-visible {
-  box-shadow:
-    inset 0 0 0 1px var(--a-surface-default),
-    var(--a-shadow-focus);
-}
+  .rdp-day_disabled {
+    cursor: not-allowed;
+    text-decoration: line-through;
+    background-color: var(--ax-bg-neutral-moderateA);
+    color: var(--ax-text-subtle);
+  }
 
-@supports not selector(:focus-visible) {
-  .navds-date .rdp-button.rdp-day_selected:not([disabled]):focus,
-  .navds-date .navds-date__month-button.rdp-day_selected:not([disabled]):focus {
-    box-shadow:
-      inset 0 0 0 1px var(--a-surface-default),
-      var(--a-shadow-focus);
+  .rdp-button:where(:not(.rdp-day_selected, [disabled])):hover,
+  .navds-date__month-button:where(:not(.rdp-day_selected, [disabled])):hover {
+    background: var(--ax-bg-accent-moderate-hover);
+    cursor: pointer;
   }
-}
 
-/* Monthpicker */
-.navds-date__month-button {
-  all: unset;
-  text-align: center;
-  width: 3rem;
-  height: 3rem;
-  text-transform: capitalize;
-  border-radius: var(--a-border-radius-medium);
-  cursor: pointer;
-}
+  .rdp-day_today {
+    box-shadow: 0 0 0 1px var(--ax-bg-accent-strong-pressed);
+    text-decoration: underline;
+  }
 
-.navds-date__month-button:focus-visible,
-.navds-monthpicker__caption-button:focus-visible {
-  box-shadow: var(--a-shadow-focus);
-  z-index: 1;
-}
+  .rdp-day_outside {
+    visibility: hidden;
+    pointer-events: none;
+  }
 
-@supports not selector(:focus-visible) {
-  .navds-date__month-button:focus,
-  .navds-monthpicker__caption-button:focus {
-    box-shadow: var(--a-shadow-focus);
-    z-index: 1;
+  .navds-date__modal& {
+    padding: 0;
+  }
+
+  .navds-date__modal-body > & {
+    padding: 0;
   }
 }
 
-.navds-date__year-label {
+.navds-date__caption {
   display: flex;
+  justify-content: space-between;
   align-items: center;
+  gap: var(--ax-spacing-1);
+  padding-inline: var(--ax-spacing-1);
 }
 
-.navds-date__wrapper,
-.navds-date__standalone-wrapper {
-  height: fit-content;
-  width: fit-content;
-}
+.navds-date__caption-button {
+  width: 2rem;
+  height: 2rem;
 
-.navds-date .navds-date__field {
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-  position: relative;
+  /* TODO: ?? */
+  &,
+  &:disabled,
+  &:hover {
+    color: var(--ax-text-default);
```

Comment:
This currently breaks hover+active on caption-buttons
<img width="55" alt="Screenshot 2024-11-25 at 14 25 02" src="https://github.com/user-attachments/assets/fcc0f499-7898-49f4-bf62-894040c59eef">